### ISSUE TITLE
Can't use contentful login

### ISSUE THREAD

**Issue by synoptase** [contentful/contentful-cli#138]
## Expected Behavior

`contentful login`
`Open a browser window now? Yes`
`Paste your token here: `

Pasting token should validate login (first time using the lib, don't know what's the exact step).

## Actual Behavior

`contentful login`
`Open a browser window now? Yes`
`Paste your token here: `
Hitting `enter` after pasting key just clears the input, no matter how many times operation is repeated.

## Steps to Reproduce

1. `contentful login`
2. `Open a browser window now? Yes`
3. `Paste your token here: `

## Context
Got a couple of warnings after installing the package:

```
npm WARN deprecated joi@13.7.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated topo@3.0.3: This module has moved and is now available at @hapi/topo. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
npm WARN deprecated hoek@5.0.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated hoek@6.1.3: This module has moved and is now available at @hapi/hoek. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
npm WARN deprecated joi@10.6.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated items@2.1.2: This module has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version of hapi to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
```

Tried downgrading `contentful-cli` to 0.16.1, without any differences.

## Environment

-   **Language Version**: node v11.9.0
-   **Package Manager Version**: npm 6.8.0
-   **Browser Version**: chrome 74.0.3729.131
-   **Operating System**: osx 10.14.4
-   **Package Version**: 0.26.4

**Comment by djohalo2** [comment]
+1

**Comment by wheldens** [comment]
+1

For now i found a workaround by adding the token in the config file.
contentful config add --mt  $TOKEN

**Comment by Khaledgarbaya** [comment]
I was able to reproduce this, it looks like something wrong with a new os update.
We'll work on a fix asap.

Cheers

**Comment by floelhoeffel** [comment]
Has this been published? I am still seeing the error in `0.26.4`

**Comment by Khaledgarbaya** [comment]
Circle CI is timing out 🤦‍♂

**Comment by fsarachu** [comment]
As a workaround you can create the `.contentfulrc.json` file manually on your working directory, with the following content:

```json
{
  "cmaToken": "paste_your_token_here"
}
```

**Comment by Khaledgarbaya** [comment]
this is now fixed in `v0.26.5`
